app/html/index.html

```diff
@@ -16,7 +34,11 @@
     <p id="loginStatus">&nbsp;</p>
   </div>
 
-  <script src="js/index.js"></script>
+  <!-- footer.html will be in here -->
+  <footer include-html="/html/footer.html"></footer>
 </body>
+<script>
```

Comment:
I'd include this script in the body similar to how the other one was.
Example: 
```
<body>

<script>
    includeHTML();
</script>
</body>
```

Reply:
Okay!